tinydir.h

```diff
@@ -704,6 +704,17 @@ if (errno)
 	base_name =basename(file_name_buf);
 #endif
 
+	/* Special case: if the path is a root dir, open the parent dir as the file */
```

Comment:
I think this is Windows specific, so it could be inside the #ifdef above

Reply:
I haven't tested this but it might work with the `/` path as well. Will need to test later

Reply:
Unfortunately, it doesn't. For `/` both `dirname` and `basename` return `/`.

Reply:
Regarding the comment:

    /* Special case: if the path is a root dir, open the parent dir as the file */

you meant the opposite (do not open the parent dir)?